Auth/OAuth2/class.FlipsideAuthenticator.php

```diff
@@ -3,11 +3,16 @@
 
 class FlipsideAuthenticator extends OAuth2Authenticator
 {
-    private $apiUrl = 'https://profiles.burningflipside.com/api/v1';
-    private $oauthUrl = 'https://profiles.burningflipside.com/OAUTH2';
```

Comment:
This change would actually break my local use of it. I use this to not have to create a local user DB and just let myself remotely authenticate. That's basically the whole point of this class. Do you have a use case for wanting this to hit your local profiles site?